x-pack/plugins/stack_connectors/public/connector_types/pagerduty/pagerduty_params.tsx

```diff
@@ -165,10 +167,17 @@ const PagerDutyParamsFields: React.FunctionComponent<ActionParamsProps<PagerDuty
                 : i18n.translate(
                     'xpack.stackConnectors.components.pagerDuty.dedupKeyTextFieldLabel',
                     {
-                      defaultMessage: 'DedupKey (optional)',
+                      defaultMessage: 'DedupKey',
```

Comment:
What do you think about removing the `isDedupeKeyRequired` logic as the label is the same for both scenarios?

Reply:
Good catch!

Reply:
Fixed in 31f46b9b70d9647d4cd8f2494bf573cb57c17d80